201-decrypt-running-linux-vm/azuredeploy.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the virtual machine"
+      }
+    },
+    "volumeType": {
+      "type": "string",
```

Comment:
If this parameter only allows a single value, you should inline that value into the extension resource's "VolumeType" property.